src/network/networkd-link.c

```diff
@@ -910,6 +918,61 @@ int link_route_remove_handler(sd_netlink *rtnl, sd_netlink_message *m, void *use
         return 1;
 }
 
+static int neighbor_handler(sd_netlink *rtnl, sd_netlink_message *m, void *userdata) {
```

Comment:
I'd like to move `neighbor_handler()` to `networkd-neightbor.c`, and make `neighbor_configure()` take `NULL` as the callback. See #10976.

Reply:
I've updated this to follow the default callback convention but not rebasing on that commit unless it gets merged before this one. I'll update the async netlink call if that gets in.